packages/mobile/ios/Podfile.lock

```diff
@@ -1604,7 +1604,7 @@ SPEC CHECKSUMS:
   ffmpeg-kit-react-native: 3cea88c9c5cfad62e1465279ea7d800dfbba3b00
   FingerprintPro: c6444f5a00d1126446da664124529e754475f198
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
-  glog: 035f1e36e53b355cf70f6434d161b36e7d21fecd
+  glog: c5d68082e772fa1c511173d6b30a9de2c05a69a2
```

Comment:
Ah turns out it might be an xcode version thing: https://github.com/facebook/react-native/issues/31121#issuecomment-1979125015